rest-service/manager_rest/resources_v2.py

```diff
@@ -300,3 +305,54 @@ def get(self, _include=None, filters=None, **kwargs):
         node_instances = get_storage_manager().get_node_instances(
             include=_include, filters=filters)
         return node_instances
+
+
+class ProviderContext(resources.ProviderContext):
+    @swagger.operation(
+        responseClass=responses.ProviderContext,
+        notes="Updates the provider context",
+        parameters=[{'name': 'global_parallel_executions_limit',
+                     'description': "the global parallel executions limit",
+                     'required': True,
+                     'allowMultiple': False,
+                     'dataType': 'int',
+                     'paramType': 'body'},
+                    ],
+        consumes=[
+            "application/json"
+        ]
+    )
+    @exceptions_handled
+    @marshal_with(responses.ProviderContext)
+    def patch(self, **kwargs):
+        """
+        modifies provider context configuration
+        """
+        verify_json_content_type()
+        request_json = request.json
+        verify_parameter_in_request_body('global_parallel_executions_limit',
+                                         request_json)
+
+        provider_ctx = get_storage_manager().get_provider_context()
+        bootstrap_ctx = provider_ctx.context['cloudify']
+
+        transient_dep_workers_mode_enabled = bootstrap_ctx.get(
+            'transient_deployment_workers_mode', {}).get(
+            'enabled', TRANSIENT_WORKERS_MODE_ENABLED_DEFAULT)
+        if not transient_dep_workers_mode_enabled:
+            raise manager_exceptions.BadParametersError(
+                "can't modify global_parallel_executions_limit since transient"
+                ' deployment workers mode is disabled')
+
+        limit = request_json['global_parallel_executions_limit']
+        if type(limit) is not int:
+            raise manager_exceptions.BadParametersError(
+                'global_parallel_executions_limit parameter should be of type'
+                ' int, but is instead of type {0}'.format(
+                    type(limit).__name__))
+
+        bootstrap_ctx['transient_deployment_workers_mode'][
```

Comment:
if you take the `get` approach suggested previously and we switch to default `True` then `transient_deployment_workers_mode` may not exist in `bootstrap_ctx`. Also, you will need to put it back into `provider_ctx.context`